protocol-methods.rst

```diff
@@ -2,6 +2,149 @@
  Protocol Methods
 ==================
 
+blockchain.address.get_balance
+==============================
+
+Return the confirmed and unconfirmed balances of a Bitcoin Cash address.
+
+**Signature**
+
+  .. function:: blockchain.address.get_balance(address)
+  .. deprecated:: 1.2 removed in version 1.3, re-added in version 1.4.3
+
+  * *address*
+
+    The address as a Cash Address (with or without prefix) or a Base58 string.
```

Comment:
Can we drop base58 support?

Reply:
Ok. How shall we word this then? 

```
The address as a Cash Address (with or without prefix).  Some server implementations MAY also support Legacy (base58) encoded addresses as well.
```

Reply:
Ok, I just pushed another commit just now.  Note there are 2 new commits now for you to review.